projects/packages/my-jetpack/_inc/data/welcome-banner/use-welcome-banner.ts

```diff
@@ -23,7 +26,8 @@ const useWelcomeBanner = () => {
 	} );
 
 	return {
-		dismissWelcomeBanner,
+		dismissWelcomeBanner: () =>
+			dismissWelcomeBanner( null, { onSuccess: () => setIsWelcomeBannerVisible( false ) } ),
```

Comment:
As a personal preference I tend to avoid defining functions in the return statement if possible, do you think it would be better to define this function in the body of the hook and then return it here?

Reply:
Good point, I got a bit upset that we need to repeat too much code here, but it will increase readability indeed.